src/NuGetPackageVerifier/Rules/PackageAuthorRule.cs

```diff
@@ -0,0 +1,36 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using NuGet.Packaging;
+using NuGetPackageVerifier.Logging;
+
+namespace NuGetPackageVerifier.Rules
+{
+    public class PackageAuthorRule : IPackageVerifierRule
+    {
+        private const string _expectedAuthor = "Microsoft";
+
+        public IEnumerable<PackageVerifierIssue> Validate(
+            FileInfo nupkgFile,
+            IPackageMetadata package,
+            IPackageVerifierLogger logger)
+        {
+            if (package.Authors == null || package.Authors.Count() < 1)
+            {
+                yield return PackageIssueFactory.RequiredAuthor();
+            }
+
+            if (package.Authors.Any(author => string.Equals(author, _expectedAuthor)))
```

Comment:
We always specify exactly one author. maybe doing `SingleOrDefault` would suffice here?